common/djangoapps/util/testing.py

```diff
@@ -55,3 +57,36 @@ def setUp(self, *args, **kwargs):
 
         self._reset_urls(urlconf_modules)
         self.addCleanup(lambda: self._reset_urls(urlconf_modules))
+
+
+class EventTestMixin(object):
```

Comment:
Thanks for the explanation. I think it is probably okay as is, because these test classes can provide more than just the patching/watching capabilities.